packages/nextjs/test/config.test.ts

```diff
@@ -286,12 +293,45 @@ describe('webpack config', () => {
 });
 
 describe('Sentry webpack plugin config', () => {
-  it('includes expected properties', () => {
-    // TODO
+  it('includes expected properties', async () => {
+    // also, can pull from either env or user config (see notes on specific properties below)
+    const finalWebpackConfig = await materializeFinalWebpackConfig({
+      userNextConfig,
+      userSentryWebpackPluginConfig,
+      incomingWebpackConfig: serverWebpackConfig,
+      incomingWebpackBuildContext: serverBuildContext,
+    });
+
+    expect(finalWebpackConfig.plugins?.[0].options).toEqual(
+      expect.objectContaining({
+        include: [
+          // because this is the server build
+          { path: '.next/server/chunks/', urlPrefix: '~/_next/server/chunks' },
+          { path: '.next/server/pages/', urlPrefix: '~/_next/server/pages' },
+          { path: '.next/serverless/', urlPrefix: '~/_next/serverless' },
+        ],
+        ignore: [], // default
+        org: 'squirrelChasers', // from user webpack plugin config
+        project: 'simulator', // from user webpack plugin config
```

Comment:
I prefer just asserting with the variable instead of just grabbing strings. Make it easier to change stuff in the future (we can also get rid of the comments then). If this works for you, we can update `release` and `dryRun` to refer to `baseBuildContext` as well.

```suggestion
        org: userSentryWebpackPluginConfig.org,
        project: userSentryWebpackPluginConfig.project,
```

Reply:
I actually had that originally. I do see the advantage you point out in terms of future changes, though I think it's six of one, half dozen of the other when it comes to saying where it's from via a variable name or via a comment. Sometimes I lean towards putting the actual values just because I find it marginally easier to grok at a glance and if things do every change, fixing it isn't hard. Don't feel strongly about it, though, so if you do, I can change it back.

Reply:
I'll let you have the final call on this one lol :uno_reverse_card: